### ISSUE TITLE
Add Resolver AS to the DNS Queries block in the measurement page

### ISSUE THREAD

**Issue by fortuna** [ooni/explorer#968]
Currently the DNS Queries block shows the resolver IP, but not the AS number and name (unlike the answers):
![image](https://github.com/user-attachments/assets/04e2db96-b06e-45f9-b479-8ed40d383b40)

This prevents me from showing that changing the Resolver can bypass DNS blocking with screenshots without annotation.

**Comment by hellais** [comment]
Yeah this probably an easy fix for measurements where it's stored inside of the JSON, yet we are planning to change these views to not have to parse any of the JSON since all these fields are already in the new DB tables.

The UI side should be considered when we get to integrating that.
